edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesRuntimeInfoProvider.cs

```diff
@@ -242,21 +238,21 @@ static RuntimeData GetRuntimedata(V1ContainerStatus status)
         ModuleRuntimeInfo ConvertPodToRuntime(string name, V1Pod pod)
         {
             Option<V1ContainerStatus> containerStatus = GetContainerByName(name, pod);
-            ReportedModuleStatus moduleStatus =  this.ConvertPodStatusToModuleStatus(containerStatus);
+            ReportedModuleStatus moduleStatus = this.ConvertPodStatusToModuleStatus(containerStatus);
             RuntimeData runtimeData = GetRuntimedata(containerStatus.OrDefault());
 
             string moduleName = name;
-            pod.Metadata?.Annotations?.TryGetValue(Constants.k8sEdgeOriginalModuleId, out moduleName);
+            pod.Metadata?.Annotations?.TryGetValue(Constants.K8sEdgeOriginalModuleId, out moduleName);
```

Comment:
I think there is a possibilty of an error here: I believe the intention is to use `name` as a `moduleName` when there is   no podName stored in annotations or annotations is empty. But I have docs say 

> When this method returns, contains the value associated with the specified key, if the key is found; otherwise, the default value for the type of the value parameter. This parameter is passed uninitialized.

So I think you need to check `TryGetValue` method return value and initialize `moduleName` appropriately

Reply:
Also agree, fixed.